### PR TITLE
SOP for adding users to dataplane osd cluster

### PR DESCRIPTION
**Pull request by R-Lawton**

### CODE REVIEW

File: sops/adding_user_to_sso.asciidoc [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+2. Make sure the users can login to data plane cluster using the kafka_sre idp option
+
+[start=3]
+3. Ensure they have the dedicated admin permissions in the cluster by checking the groups in the user managment tab of the cluster
+
+
+== Troubleshooting
+
+If people are unable to login ensure the username specified in sso in the same as the one your giving permissions to in user.json
```
Comment (obrienrobert) [inline sops/adding_user_to_sso.asciidoc:60]: `in sso in the same` -> `in sso is the same`

File: sops/adding_user_to_sso.asciidoc [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+== Execute/Resolution
+=== Creating the user in SSO
+1. Login into MAS SSO
+2. Navigate to the mas_kafka_sre realm and then to users in the side bar on the left.
+3. Create the new user by clicking on the add user button on the right hand side
+4. Give the user a user name and any other information you want to add. Make sure the user enabled slider is set to on.
+
+image::images/add_user.png[]
+[start=5]
+5. Once you click save you will be able to give the user credentials by clicking the credentials tab and either giving a temporary password which the users changes when the first login or a permanent password that cant be changed later when the user logs in.
```
Comment (obrienrobert) [inline sops/adding_user_to_sso.asciidoc:40]: `giving` -> `given`
`when the first login` -> `when they first login`

File: sops/adding_user_to_sso.asciidoc [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+image::images/add_creds.png[]
+
+
+=== giving the user permission in OSD data plane cluster
+1. To give the users permission you will need ocm permissions, user.json and ocm post command which can be gotten from the SOP document in the onboarding RTS doc
+
+== Validate
+
+1. Check that the users created using the ocm get command from the SOP doc.
```
Comment (obrienrobert) [inline sops/adding_user_to_sso.asciidoc:50]: @R-Lawton tiny little suggestions:

Check that the users `are` created. Or maybe Check that the users `have` been created

File: sops/adding_user_to_sso.asciidoc [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+2. Make sure the users can login to data plane cluster using the kafka_sre idp option
```
Comment (obrienrobert) [inline sops/adding_user_to_sso.asciidoc:52]: log in to `the` data plane cluster

File: sops/adding_user_to_sso.asciidoc [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+== Description
+
+The purpose of this SOP is to describe the process of creating users in MAS-SSO for Data Plane OSD cluster access
+
+
+== Prerequisites
+
+You will need access to the following:
+
+* mas_kafka_sre realm in the MAS-SSO in the MAS-SSO cluster
```
Comment (obrienrobert) [inline sops/adding_user_to_sso.asciidoc:26]: mas_kafka_sre realm in the MAS-SSO in the MAS-SSO cluster ->  mas_kafka_sre realm in the MAS-SSO cluster

File: sops/adding_user_to_sso.asciidoc [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+=== Creating the user in SSO
+1. Login into MAS SSO using the credentials from the URL doc in the RTS getting started guide
+2. Navigate to the mas_kafka_sre realm and then to users in the side bar on the left.
+3. Create the new user by clicking on the add user button on the right hand side
+4. Give the user a user name and any other information you want to add. Make sure the user enabled slider is set to on.
+
+image::images/add_user.png[]
+[start=5]
+5. Once you click save you will be able to give the user credentials. This is done by by clicking the credentials tab and either giving them a temporary password which the user changes when they first login or a permanent password that can't be changed.
+image::images/add_creds.png[]
```
Comment (obrienrobert) [inline sops/adding_user_to_sso.asciidoc:41]: This image doesn't seem to be appearing for me now (maybe it's a formatting error?)

Comment (R-Lawton) [reply sops/adding_user_to_sso.asciidoc:42]: Oh good catch, fixed now

File: sops/adding_user_to_sso.asciidoc [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+== Execute/Resolution
+=== Creating the user in SSO
+1. Login into MAS SSO using the credentials from the URL doc in the RTS getting started guide
+2. Navigate to the mas_kafka_sre realm and then to users in the side bar on the left.
+3. Create the new user by clicking on the add user button on the right hand side
+4. Give the user a user name and any other information you want to add. Make sure the user enabled slider is set to on.
+
+image::images/add_user.png[]
+[start=5]
+5. Once you click save you will be able to give the user credentials. This is done by by clicking the credentials tab and either giving them a temporary password which the user changes when they first login or a permanent password that can't be changed.
```
Comment (obrienrobert) [inline sops/adding_user_to_sso.asciidoc:40]: @R-Lawton looks good. Just noticed one more tiny little type: by by

Comment (R-Lawton) [reply sops/adding_user_to_sso.asciidoc]: Genie how did i miss that, thats pushed up thanks Rob

**Comment by obrienrobert** [comment]
@R-Lawton looks good to me.

**Pull request merged**